helper/funcs.go

```diff
@@ -461,3 +462,22 @@ func RemoveEqualFold(xs *[]string, search string) {
 		}
 	}
 }
+
+// CheckNamespaceScope ensures that the provided namespace is equal to
+// or a parent of the requested namespaces
```

Comment:
Describe the return value:
```suggestion
// or a parent of the requested namespaces. Returns requested namespaces which are not equal to or a child of the provided namespace.
```